frontend/src/metabase/query_builder/components/notebook/steps/SortStep/SortStep.styled.tsx

```diff
@@ -12,7 +10,3 @@ export const SortDirectionButton = styled.button`
   font-weight: 700;
   cursor: pointer;
 `;
-
-export const SortColumnPicker = styled(QueryColumnPicker)`
-  color: ${color("summarize")};
-`;
```

Comment:
Aggregations and breakouts use green "summarize" colors, the sort step should be grey (default)